metrics/metrics.go

```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+//Provider is intended to be a metrics API for webpa servers.
+type Provider struct {
+	DefaultGathererInUse bool
+}
+
+//GetCounter returns a Counter metrics Collector
+//If MetricsTool.DefaultGathererInUse is true, it will go ahead and do the registration for such metric
+//with the prometheus defaultGatherer
+func (m *Provider) GetCounter(name, help string, labelValues []string) (counter metrics.Counter) {
```

Comment:
Again, this function isn't necessary for the first version of this package.  Just use the providers in go-kit's `metrics.provider` package.